docs/index.html

```diff
@@ -269,7 +269,7 @@ <h3 class="padding-left-small padding-bottom-large">About</h3>
         <div class="pt-card-large">
           <h4>What</h4>
           <h3 class="b-header pt-header-card pt-text-card">Onboarding material for license implementation</h3>
-          <p class="text-normal">This toolkit brings all the elements for a basic Creative Commons platform integration. </p>
+          <p class="text-normal">This toolkit brings all the elements for a basic Creative Commons platform integration.</p>
```

Comment:
@renuka142 Thanks for your contribution! 
Plzzz revert this unnecessary change of linting. Make it in accordance with the complete repo.